api/events/lambda_function.py

```diff
@@ -55,7 +55,12 @@ def lambda_handler(event, _):
         if event["source"] == "aws.medialive":
             # handle medialive pipeline alerts
             if "MediaLive" in event["detail-type"] and "Alert" in event["detail-type"]:
-                event["resource_arn"] = event["detail"]["channel_arn"]
+                # multiplex pipeline alerts do not have a "detail.channel_arn" property.
+                if "Multiplex" in event["detail-type"]:
+                    event["resource_arn"] = event["detail"]["multiplex_arn"]
+                    item["resource_arn"] = event["detail"]["multiplex_arn"]
```

Comment:
We can eliminate this line if we add "multiplex_arn" to line 42.

Reply:
Perfect will make that update shortly.